domains/s.drpleaserespect.json

```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "DrPleaseRespect",
+    "email": "juliannayr2007@gmail.com"
+  },
+  "record": {
+    "A": ["146.19.100.135"],
+    "AAAA": ["2a05:dfc1:4400:6c00::a"]
```

Comment:
Inaccessible site.

Reply:
Currently using a reverse proxy for this, nothing is bounded to the raw IP. I used the same DNS Records as my other sub-subdomain "server.drpleaserespect.is-a.dev". If you want I can temporarily open a random port to open the site temporarily (without HTTPS)

Reply:
Here it is "404" cause it there's no redirect at main page. and the controls are locked behind my other sub-subdomain
http://146.19.100.135:4500/